socialhome/streams/app/components/Stream.vue

```diff
@@ -71,6 +71,16 @@ export default Vue.component("stream", {
                 console.error(`Unsupported stream name ${this.$store.state.streamName}`)
             }
         },
+        translations() {
+            return {
+                newPostsAvailables: interpolate(
```

Comment:
`newPostsAvailables` -> `newContentAvailable`.

Reply:
We should avoid using globals (interpolate) when we can. How about `_.template` which can be imported? https://lodash.com/docs/4.17.4#template

```
>>> var t = _.template(ngettext("<%= ids %> new post available", "<%= ids %> new posts available", 3))
>>> t({ids: 5})
"5 new posts available"
```

Reply:
Why not?

Reply:
You mean why avoid? If there are options, the components should not IMHO rely on global javascript being available. This `interpolate` comes from Django right? Maybe at some point we want to get rid of the Django dependency (`gettext`). At that point, it's more work to convert these places. So as far as we're able to, let's make components that don't depend on magically available code.

But, thinking about this for a while, in this particular case maybe `interpolate` is fine. As long as we only use it when it's not so easy to use standard JS stuff or proper templates like `_.template`.

Reply:
> Why not?

No, I meant: "*why not use `_.template`*?". I was basically agreeing ;)

Reply:
I used JS' template strings in the end.